fixtures/templateloader.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Preloads all templates on all providers that were selected for testing. Useful for test collect.
+"""
+import pytest
+
+from fixtures.prov_filter import filtered
+from fixtures.terminalreporter import reporter
+from utils.providers import provider_factory
+
+TEMPLATES = {}
+
+
+@pytest.mark.trylast
+def pytest_configure(config):
+    terminal = reporter()
+    terminal.write("Loading templates from providers:\n")
```

Comment:
I don't think this needs to write out to the terminal unless something is wrong.

When something is wrong, [write_line(msg, red=True)](https://github.com/RedHatQE/cfme_tests/blob/master/fixtures/pytest_store.py#L148) will save you a few lines.